packages/core/src/Stores/common-store.js

```diff
@@ -54,7 +59,11 @@ export default class CommonStore extends BaseStore {
 
     @action.bound
     setPlatform() {
-        this.platform = new URL(window.location).searchParams.get('platform');
+        const search = window.location.search;
+        if (search) {
+            const url_params = new URLSearchParams(search);
+            this.platform = url_params.get('platform') || '';
+        }
```

Comment:
(platform-context) (throughout) Why is this introduced as a store variable, we can use `PlatformContext` instead. Unless, is this something different than what we offer in PlatformContext?

Reply:
The `platform` store variable is used to toggle some mobile-app-origin features (hide from banner + redirect back to mobile app, etc). This state was here before this PR, so for consistency sake I just extended using these states. Hope that makes sense 🙏🏼  @reza-fs